modules/update-using-custom-machine-config-pools-about.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * updating/update-using-custom-machine-config-pools.adoc
+
+[id="update-using-custom-machine-config-pools-about_{context}"]
+= About performing a canary rollout update
+
+This topic describes the general workflow of this canary rollout update process. The steps to perform each teak in the workflow are decribed in the follwoing sections.
+
+. Create MCPs based on the worker pool. The number of nodes in each MCP depends on a few factors, such as your maintaince window duration for each MCP, and the amount of reserve capacity (extra worker nodes) available in your cluster.
++
+[NOTE]
+====
+You can change the `maxUnavailable` setting in a MCP to specify the percentage or the number of machines that can be updating at any given time. The default is 1.
```

Comment:
I did not find `maxUnavailable` from a mcp yaml file got from `oc get` cmd. @rioliu-rh Could you help review this part about the `maxUnavailable` setting of a MCP, thanks!